### ISSUE TITLE
Inclusion of SNLI test dataset to find accuracies in the base LSTM entailment model

### ISSUE THREAD

**Issue by SwagatSBhuyan** [dh1105/Sentence-Entailment#3]
As of now, the model is simply trained and a set of training accuracy and validation accuracy is recorded after the epochs in the base LSTM model. Can you also try to integrate the test snli dataset so that a better understanding of the entailment scores can be recorded?
